### ISSUE TITLE
Database design

### ISSUE THREAD

**Issue by qy21gafy** [amosproj/amos2022ws05-shared-desk-mgmt#33]
## User story
1. As a software developer
2. I want / need ma design for my database
3. So that I can initialize it for the project

## Acceptance criteria
* Database design should consider user data (name, company, permissions, admin status etc.)
* Database design should consider booked desks of each user (desk, time, place, user etc.)
* Database design should consider available resources of each company (name, type, place, capacity etc.)
* Database design should consider different company data (e.g name, country, locations, buildings etc.)
* Database design should be normalized

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
